packages/playwright-core/types/har.d.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// see http://www.softwareishard.com/blog/har-12-spec/
+export type HAR = {
+  log: HARLog;
+}
+
+export type HARLog = {
+  version: string;
+  creator: HARCreator;
+  browser: HARBrowser;
```

Comment:
Let's align with the spec. For example, `browser` and `pages` are optional, and we also lack the `comment` field. Similar issues in other types.